test/test_common/utility.h

```diff
@@ -95,7 +95,7 @@ class TestUtility {
 class ConditionalInitializer {
 public:
   /**
-   * Set the conditional to ready, should only be called once.
+   * Set the conditional to ready.
```

Comment:
Maybe explain that `waitReady()` resets, but in between `waitReady()`, `setReady()` should only be called once.